### ISSUE TITLE
orb_v1() got an unexpected keyword argument 'device'

### ISSUE THREAD

**Issue by wuziqiqiqi** [orbital-materials/orb-models#19]
I was running 
```
from orb_models.forcefield import pretrained
orbff = pretrained.orb_v1(device='cuda')
```
and get an error saying `orb_v1() got an unexpected keyword argument 'device'`
I was running it on Linux and with torch 2.2.0+cu121. Any suggestions on what might be the problem?

**Comment by wuziqiqiqi** [comment]
doing
```
import torch
torch.set_default_device('cuda')
orbff = pretrained.orb_v1()
```
seems to mitigate the issue, but it still weird that I can't do `orbff = pretrained.orb_v1(device='cuda')`

p.s. This is a recent installation using 
```
pip install orb-models
pip install "pynanoflann@git+https://github.com/dwastberg/pynanoflann#egg=af434039ae14bedcbb838a7808924d6689274168",
```

**Comment by benrhodes26** [comment]
This is fixed in the latest version 0.3.2. Could you try `pip install --upgrade orb_models`?
